VK/VK/UserPhotosCollection/VC/UserPhotosCollectionViewController.swift

```diff
@@ -42,12 +40,23 @@ final class UserPhotosCollectionViewController: UICollectionViewController {
 
     // MARK: - Private Methods
 
+    private func loadData() {
+        fetchData()
+        guard let savedPhotos = dataBaseService.loadData(objectType: Photo.self) else { return }
+        photos = savedPhotos.filter { $0.ownerID == self.userID }
+        collectionView.reloadData()
+    }
+
+    private func saveData(data: [Photo]) {
```

Comment:
```suggestion
    private func saveData(photos: [Photo]) {
```
Дзіма гэта не data называй зменныя гэтак жа як і іх тыпы